ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -257,6 +263,17 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         this.initializeComponentState();
     }
 
+    private setupScrollToErrorAction(): void {
+        this.scroll_up$ = this.validationRequested$.pipe(
+            map(validationRequested => validationRequested && this.block.scrollTo),
+            debounceTime(300)
+        );
+    }
+    scrollUpExecutedAction(executed: boolean): void {
+        this.block.scrollTo=!executed;
+        this.validationRequested$.next(!executed);
```

Comment:
I believe we should not trigger validationRequested$ flag here.
The main logic with `validationRequested$` is in activity.component related to Next/Submit buttons click behavior.

I guess, the reason you added it is a debounceTime on line 269.
The debounce prevents the scroll-up directive working, can be removed.

Reply:
So, eventually it should be like:

```
  private setupScrollToErrorAction(): void {
        this.scroll_up$ = this.validationRequested$.pipe(
            map(validationRequested => validationRequested && this.block.scrollTo),
        );
  }

  scrollUpExecutedAction(): void {
      this.block.scrollTo = false;
  }


```

Reply:
done